modules/fedimint-ln-server/src/lib.rs

```diff
@@ -943,14 +953,39 @@ impl ServerModule for Lightning {
                     Ok(())
                 }
             },
+            api_endpoint! {
+                REMOVE_GATEWAY_CHALLENGE_ENDPOINT,
+                ApiVersion::new(0, 0),
```

Comment:
This isn't true, will only be available on next minor API version (also need to bump it)

Reply:
Bumped the `ApiVersion` here and above in `supported_api_version`